File: ts/src/test/base/test.stringToBase64.ts


import assert from 'assert';
import ccxt from '../../../ccxt.js';

function testStringToBase64 () {

    const exchange = new ccxt.Exchange ({
        'id': 'sampleexchange',
    });

    assert (exchange.stringToBase64 ('hello world 123!@#$%^&*()"\'-+)S') === 'aGVsbG8gd29ybGQgMTIzIUAjJCVeJiooKSInLSspUw==');
}

export default testStringToBase64;

File: julia/Ccxt/test/base/test.stringToBase64.jl
using Test
using Ccxt
function testStringToBase64()

    exchange = get(ccxt, Symbol("Exchange"), nothing)(Dict{Symbol, Any}(
        Symbol("id") => "sampleexchange"
    ));
    @test stringToBase64(exchange, "hello world 123!@#\$%^&*()\"-+)S") == "aGVsbG8gd29ybGQgMTIzIUAjJCVeJiooKSItKylT";
end
